pages/common/zlib-flate.md

```diff
@@ -0,0 +1,17 @@
+# zlib-flate
+
+> Raw zlib compression and decompression program.
+> The zlib-flate program is part of the qpdf package.
+> More information: <https://manned.org/zlib-flate>.
+
+- Compress a file:
+
+`zlib-flate -compress < {{file}} > {{compressed.zlib}}`
+
+- Uncompress a file:
+
+`zlib-flate -uncompress < {{compressed.zlib}} > {{file}}`
```

Comment:
```suggestion
`zlib-flate -uncompress < {{path/to/compressed.zlib}} > {{path/to/input_file}}`
```

Reply:
Thanks for the suggestions!
In this case i think `output_file` is more appropriate. Already updated.